pkg/didcomm/common/model/didexchange/models.go

```diff
@@ -0,0 +1,20 @@
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
```

Comment:
Why isn’t this part of client package

Reply:
it creates cyclic dependency as the implementation is present in [service pkg](https://github.com/hyperledger/aries-framework-go/pull/616/files#diff-79de382a052bbdb4213b9dd373b0e2f6R15). Also, @fqutishat and I were thinking that the [common models](https://github.com/hyperledger/aries-framework-go/blob/master/pkg/client/didexchange/models.go#L47) (currently client models which wrap service models) between client and service could be moved to this common package.